account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
```

Comment:
Can you elaborate the scenario that causes problems for you? In my understanding credit or debit can never be negative in Odoo.

So we have the following:
* supplier invoice: credit > 0 on payable account
* supplier refund: not something to be paid on a credit transfer order
* customer refund: not something to be paid on a credit transfer order
* customer refund: credit > 0 on receivable account